### DISCUSSION TITLE
layui.dev在海外无法正常加载

### DISCUSSION THREAD

**Discussion by Hefulalala** [layui/layui#2338] [问答]
![image](https://github.com/user-attachments/assets/3dff3c64-4717-47bd-beb0-af79d13aad08)
![image](https://github.com/user-attachments/assets/2e4bf4fc-560a-4392-86fc-49786ab3af94)

**Comment by sentsim** [comment]
现在是否正常了呢？

**Reply by Hefulalala** [reply]
正常了
